app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="<%= dom_id post %>" class="user_con">
+    <div class="user_data_post">
+     <%= link_to "#{post.title}", user_post_path(@user.id, post.id), class:"anything" %> 
+      <h3><%= post.text%></h3>
+      <p>Number of comments: <%= post.comments_counter %> <span>Number of likes: <%= post.likes_counter %></span></p>
+    </div>
+
+  </div>
+
+  <div class="comments">
+    <% post.recent_5_comments.each do |comment| %>
+      <h4><%= comment.author.name%>: <%= comment.text%></h4>
+    <%end%>
```

Comment:
- [ ] I am getting an error here,  because you don't have `recent_5_comments`, you have `last_five_comments` kindly fix the spelling